### PR TITLE
Add location support

### PR DESCRIPTION
**Pull request by wannes-ds**
This PR adds location configuration to support a non-local time zone

### CODE REVIEW

File: cron.go [hunk 1]
```diff
@@ -71,13 +72,19 @@ func (s byTime) Less(i, j int) bool {
 
 // New returns a new Cron job runner.
```
Comment (robfig) [inline cron.go]: A small update would probably be helpful:

// New returns a new Cron job runner, in the Local time zone.

**Comment by robfig** [comment]
Sorry for the delay! This looks great, just one tiny tweak

**Comment by wannes-ds** [comment]
Done :)

**Comment by robfig** [comment]
Github says it can't be merged?

**Comment by wannes-ds** [comment]
Some merge conflicts occurred, should be mergeable now

File: cron.go [hunk 2]
```diff
@@ -195,7 +207,7 @@ func (c *Cron) run() {
 		}
 
 		// 'now' should be updated after newEntry and snapshot cases.
-		now = time.Now().Local()
+		now = time.Now().In(c.location)
```
Comment (robfig) [inline cron.go]: I see a remaining time.Now().Local() on line 199 -- seems like that should also get updated?

**Comment by wannes-ds** [comment]
Ah, missed one. Fixed, thanks

**Comment by robfig** [comment]
Thank you!

**Pull request merged**